lib/plist-service/index.js

```diff
@@ -28,10 +33,23 @@ class PlistService {
   }
 
   async receivePlist (timeout = 5000) {
-    this._splitter.resume();
     return await new B((resolve, reject) => {
-      setTimeout(function () { reject(new Error(`Failed to receive any data within the timeout: ${timeout}`)); }, timeout);
-      this._decoder.once('data', (data) => resolve(data));
+      const queue = this.replyQueue;
+      const data = queue.shift();
+      if (data) {
+        resolve(data);
+      }
+      const checkExist = setInterval(() => {
```

Comment:
Should this be happening even if there was already data?

Reply:
Good catch. I should put a return before